docs/user guide/main.tex

```diff
@@ -0,0 +1,133 @@
+\item If the file contains a first row that functions as a header, check the "First row as table header" checkbox so the no names have to be given to columns. 
+\item To specify a new column, fill in the properties of the column and press the "Add column" button. Do this for each column in the datafile. The columns in the datafile must be in the same order as specified in the column specification. If the type of the column is a temporal value, it is necessary to specify a format for the date before adding the column specification.
+\item If you make a mistake and want to change order of columns, the "Up" and "Down" buttons can be used to control the selected column specification and the "Remove" button removes the selected column specification.
+
+The next three steps are only necessary for specifying a plain text file. These are in the "Plain Textfile Specification" area.
+\item Starting line: The starting line of the file is the line at which the first column of significant data starts. So if the plain textfile contains a header, this can be skipped by specifying the starting line.
+\item Remove from end: This is the amount of lines NOT to read as data counted from the end of the plain textfile. This can for example be a footer.
+\item The last thing we need to specify is the delimiter of the file. The columns in the plain textfile must be split by some character. This character must be specified here. These can for example be a comma (,) or a tab character ($\backslash$t) etcetera.
+\\\\
+
+OPTIONAL: Specifying a filename column. In the "Add filename column" area you can choose if you want to add a column to the selected datafile specification. This column must have a name so you can reference to this in future analyses. The type of this column must also be specified (same here: if the type is a temporal value, the format must be specified). All the values of this column will be filled with the same value, namely the name of the file without the extension. This column can be used later to specify a connection between different tables based on this key value. It is also possible to specify your own value for this column instead, by filling in the "Column value" field. Don't forget to apply your changes after modifying these settings for a datafile.
+\\\\
+
+Tip: If you have multiple files with the same specification, you can copy a template of this file by selecting the file and pressing the "Copy template" button. After this is done you can apply the copied specification by selecting the target datafile and pressing the "Paste template" button.
+\\\\
+
+After all the specifications are correctly set we can create the XML file. Choose the path to save the XML file and after that the wizard will close and the program will start loading the datafiles.
+
+It is also possible to edit a previously created XML file using the XML wizard. This is done in the wizard by going to File -> Load and then choose the XML file.
```

Comment:
->  is veranderd naar een omgekeerde vraagteken, de > tussen $ fixt dat volgens mij of gebruik $\rightarrow$